frontend/src/metabase/visualizations/visualizations/Map.jsx

```diff
@@ -241,6 +246,9 @@ export default class Map extends Component {
           .map(([key, value]) => ({ name: value.name || "", value: key }))
           .sortBy(x => x.name.toLowerCase())
           .value(),
+        placeholder: t`Select a region`,
+        footerFn: () => <CustomMapFooter />,
```

Comment:
feel like if you don't need to pass anything to `<CustomMapFooter />` maybe you could just do `footer` as Alexander P. suggested. Or just name this `FooterComponent` and use it like a React component since this is also technically a React component.